graylog2-server/src/main/java/org/graylog/plugins/views/search/export/ExportMessagesCommand.java

```diff
@@ -40,6 +40,8 @@ public abstract class ExportMessagesCommand {
     public static final ElasticsearchQueryString DEFAULT_QUERY = ElasticsearchQueryString.empty();
     public static final Set<String> DEFAULT_STREAMS = ImmutableSet.of();
     public static final LinkedHashSet<String> DEFAULT_FIELDS = linkedHashSetOf("timestamp", "source", "message");
+
+    public static final LinkedHashSet<String> ALL_FIELDS = new LinkedHashSet<>();
```

Comment:
Why this semantic and not the obvious '!command.fieldsInOrder().isEmpty()` or `.size() == 0` etc.? 
Maybe create a method with a speaking name (e.g. `isSubsetOfFields` or `notAllFieldsQuery` or whatever so that the `if` is more readable? Comparing to an empty set looked strange to me.